component/admin/controllers/order_detail.php

```diff
@@ -201,9 +201,10 @@ public function update_discount()
             $msg = JText::_('COM_REDSHOP_DISCOUNT_UPDATED');
```

Comment:
@hoang-tran-redweb : please also set $type = 'success' for of coming `else` condition